botocore/config.py

```diff
@@ -183,6 +183,12 @@ class Config:
         endpoint resolution.
 
         Defaults to None.
+
+    :type tcp_keepalive: bool
+    :param tcp_keepalive: Toggles the TCP Keep-Alive socket option used when
```

Comment:
After thinking some more about this, there is actually a difference and it has to be "Setting to True enables...". Because the reverse is not true: Setting to False does not disable TCP keepalive if it was previously on for whatever reason.